go/vt/vtgate/vindexes/vschema.go

```diff
@@ -175,24 +175,32 @@ type KeyspaceSchema struct {
 	Error    error
 }
 
+type ksJSON struct {
+	Sharded  bool              `json:"sharded,omitempty"`
+	Tables   map[string]*Table `json:"tables,omitempty"`
+	Vindexes map[string]Vindex `json:"vindexes,omitempty"`
+	Views    map[string]string `json:"views,omitempty"`
+	Error    string            `json:"error,omitempty"`
+}
+
 // MarshalJSON returns a JSON representation of KeyspaceSchema.
 func (ks *KeyspaceSchema) MarshalJSON() ([]byte, error) {
-	return json.Marshal(struct {
-		Sharded  bool              `json:"sharded,omitempty"`
-		Tables   map[string]*Table `json:"tables,omitempty"`
-		Vindexes map[string]Vindex `json:"vindexes,omitempty"`
-		Error    string            `json:"error,omitempty"`
-	}{
+	ksJ := ksJSON{
```

Comment:
Is there no test for this JSON output, because I don't see any corresponding test change?

Reply:
I tested it manually, I will add a test for it.